src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs

```diff
@@ -95,6 +96,38 @@ internal async Task ProcessHttpInAndOutInvocationRequest(ScriptInvocationContext
             }
         }
 
+        private async Task<HttpResponseMessage> SendInvocationRequestAsync(ScriptInvocationContext scriptInvocationContext, HttpRequestMessage httpRequestMessage)
+        {
+            // Only log Request / Response when running locally
+            if (_environment.IsCoreTools())
+            {
+                scriptInvocationContext.Logger.LogTrace($"Invocation Request:{httpRequestMessage}");
+                await TraceHttpContent(httpRequestMessage.Content, scriptInvocationContext.Logger);
+            }
+            _logger.LogDebug("Sending http request for function:{functionName} invocationId:{invocationId}", scriptInvocationContext.FunctionMetadata.Name, scriptInvocationContext.ExecutionContext.InvocationId);
+            HttpResponseMessage invocationResponse = await _httpClient.SendAsync(httpRequestMessage);
+            if (_environment.IsCoreTools())
```

Comment:
>_environment.IsCoreTools() [](start = 16, length = 26)

Also it might be a good idea to cache this into a variable rather than calling "_environment.IsCoreTools()"  twice

Reply:
good idea. Done!